internal/dcache/replication_manager/replication_manager.go

```diff
@@ -428,8 +443,12 @@ retry:
 			if targetNodeID == rpc.GetMyNodeUUID() {
 				// Only one component RV can be local.
 				common.Assert(putChunkDCReq.Request.Chunk.Address.RvID == "",
-					putChunkDCReq.Request.Chunk.Address.String())
+					putChunkDCReq.Request.Chunk.Address.String(), rv.Name, rvID)
+				common.Assert(!isLocalRV,
```

Comment:
the prev assert is (slightly) better as it's asserting RvID is not set before writing to it.

Reply:
you are saying two different things here?
Two different blobfuse processes trying to read each others' chunks?
That's not a valid scenario. We can have two different blbofuse processes but only when that node is taking part in two different caches, in which case it'll not try to access RVs from other process.
If they have to access their chunk only, the check works fine.